retirement/migrations/0014_auto_20190730_0910.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 2.0.8 on 2019-07-30 13:10
+
+import datetime
+from django.db import migrations, models
+import django.utils.timezone
+from django.utils.timezone import utc
+
+
+def update_inscription_date(apps, schema_editor):
+    reservation_model = apps.get_model('retirement', 'reservation')
+    for reservation in reservation_model.objects.all():
+        if reservation.order_line:
+            reservation.inscription_date = reservation.order_line.order.transaction_date
+        else:
+            reservation.inscription_date = reservation.retreat.start_time
```

Comment:
I think it could be nice to put `inscription_date` one minute minimum before `start_time`, just to prevent further problem if one day we have a condition on these field. Like if the user reserved at last minute, but not at the exact second like a superhero :p